### DISCUSSION TITLE
Can Linefilters be saved with Project

### DISCUSSION THREAD

**Discussion by gemse007** [WinMerge/winmerge#1016] [Q&A]
I use WinMerge in several occasions. And in some of them I use File- and Linefilters and save the Project, so I can easily compare the two folders over and over again.
The Problem I have is that Linefilters are not saved with the Project. So when I use it to compare let's say some textfiles I don't need the Filters from above described project and I disable them.
When I load the project they are still disabled and I get lot's of differences until I recognise that I forgot to activate Linefilters.

Is there a possability to automatically set Linefilters when a project is loaded?
If not - maybe you could add this as a Featurerequest.

**Comment by sdottaka** [comment]
I copied this discussion to GitHub issues.
